apps/web/tests/e2e/fixtures/auth.ts

```
@@ -0,0 +1,43 @@
+
+export type TestUser = {
+  email: string;
+  password: string;
+  id?: string;
+};
+
+export const test = base.extend<{
+  authenticatedPage: Page;
+  testUser: TestUser;
+}>({
+  testUser: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)